drivers/spi/Kconfig.esp32

```diff
@@ -0,0 +1,34 @@
+# ESP32 SPI configuration
+
+# Copyright (c) 2020 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ESP32_SPIM
+	bool "ESP32 SPI Master driver"
+	depends on SOC_ESP32
+	default y
+	help
+	  Enables support for ESP32 SPI Master driver.
+
+if ESP32_SPIM
+
+config SPI_ESP32_HALFDUPLEX
+	bool "Half-duplex mode"
+	default n
+	help
+	  Enables half-duplex communication mode.
+
+config SPI_ESP32_NO_DUMMY
+	bool "ESP32 SPI dummy compensation"
+	default n
+	help
+	  Enables dummy SPI compensation cycles.
+
+config SPI_ESP32_3WIRE
+	bool "3-wire operation mode"
+	default n
+	help
+	  Enables 3-wire mode.
```

Comment:
The default for kconfig is `n` for `bool`, so there is really no need for `default n`.

Reply:
Yes, I have fixed that already. Will push all resolved conversation soon.